### DISCUSSION TITLE
upload soundfronts?

### DISCUSSION THREAD

**Discussion by XDKaioden** [ryohey/signal#94] [Ideas]
Add a feature where you can upload your own soundfronts

**Comment by ryohey** [comment]
I'm using a forked version of sf2synth.js, but there are performance issues, so I'm considering implementing my own module. In that module In that module I'm currently implementing, I'm thinking of using a file converted from a SoundFont to optimize load times and such.
For now, the goal is to have it work well with the default tones. I'm not sure when I'll be able to start working on this, but I'd like to consider a system that allows users to use files converted from SoundFont.

**Comment by Endermaster3690** [comment]
now what im bout to be talking about isnt about uploading, just a thing that's there from the start. how about a checkmark in the settings that makes it the default acoustic sf used when playing midis?
